tests/python_eol/patch_python_venv.sh

```diff
@@ -18,15 +18,20 @@
 #
 set -euo pipefail
 
-if [[ "$#" -ne 1 || ! "$1" =~ ^2\.6|3\.4$ ]]; then
-  echo "Usage: patch_python_venv.sh 2.6|3.4"
+if [[ "$#" -ne 1 || ! "$1" =~ ^2\.6|2\.7|3\.4$ ]]; then
+  echo "Usage: patch_python_venv.sh 2.6|2.7|3.4"
```

Comment:
2.7 left here for container patching?

Reply:
yes, for using the 2.7 venv in a container... more patching may be needed, too (I'm getting some warnings about hashlib, even though we do not use it) and 2 test failures (not sure if the failures are related to those warnings, i'll check that later)